src/Sylius/Bundle/ApiBundle/Resources/config/api_resources/ZoneMember.xml

```diff
@@ -34,6 +34,15 @@
             </itemOperation>
         </itemOperations>
 
+        <subresourceOperations>
+            <subresourceOperation name="api_zones_members_get_subresource">
```

Comment:
In the zones file, there is no configuration for this subresource operation, why?